cmd/dcrdata/views/block.tmpl

```diff
@@ -46,7 +46,11 @@
 						{{- else }}
 						<a class="fs13" href="/mempool">mempool </a>|
 						{{- end}}
+						{{if .MainChain}}
 						<a class="fs13" href="/api/block/{{.Height}}/verbose?indent=true" data-turbolinks="false">api</a>
+						{{- else}}
+						<a class="fs13" href="/api/block/hash/{{.Hash}}/verbose?indent=true" data-turbolinks="false">api</a>
```

Comment:
How about hash in both cases?

Reply:
The user would proly look at the URL to note the block they are viewing, I think we'd want to make this exception just for side blocks unless you want us to use hash for both.